.changeset/fifty-roses-brake.md

```diff
@@ -0,0 +1,7 @@
+---
+'@twilio-paste/avatar': patch
+'@twilio-paste/website': patch
```

Comment:
Hmm we typically don't patch the website. Do you think we should start?

Reply:
Ah good catch - I don't think patching the website is necessary. Just an oversight.